src/Server.php

```diff
@@ -350,7 +333,7 @@ private function handleSparqlUpdate(Response $response, string $path, $contents)
 
             if ($success) {
                 $this->removeLinkFromMetaFileFor($path);
-                $this->sendWebsocketUpdate($path);
+                $this->sendNotificationUpdate($path, "Update");
```

Comment:
It might be worth placing the notification types in a class constant ("Update", "Create", "Delete").

This would also make it more clear which types are expected/available.

Reply:
good suggestion, added in e2bd63594caa7f29bfd9d6c3699a6e921655d2b2